src/api/process_status.js

```diff
@@ -74,17 +74,62 @@ export async function processStatus(textbookRef, textbook) {
     throw new Error(`Could not find 'textbook_events' subcollection reference on textbook id: ${textbook.id}`)
   }
 
-  const textbookEvents = (await getDocs(query(textbookEventsRef, orderBy('timestamp')))).docs;
+  const textbookEvents = (await getDocs(query(textbookEventsRef, orderBy('timestamp', "desc")))).docs;
 
   if (!textbookEvents || textbookEvents.length === 0) {
     throw new Error(`Could not find 'textbook_events' subcollection on textbook id: ${textbook.id}`)
   }
 
-  // now add the logic to map the events to statuses
+  const textbookEventsData = textbookEvents.map(event => event.data());
+  const textbookStatus = evaluateEventStatus(textbookEventsData);
 
   return {
     ...textbook,
-    status: EventStatus.ACTIVE
+    status: textbookStatus
+  }
+  
+}
+
+
+const RESERVATION_EXPIRATION = 60 * 60 * 24 * 7 // one week in seconds
```

Comment:
Let's encode this as two consts so we can remove the comment

`
const ONE_WEEK_SECONDS=60*60*24*7
const RESERVATION_EXPIRATION=ONE_WEEK_SECONDS
`

Reply:
sure